sklearn/compose/tests/test_column_transformer.py

```diff
@@ -2447,6 +2448,26 @@ def test_column_transformer_error_with_duplicated_columns(dataframe_lib):
         transformer.fit_transform(df)
 
 
+def test_column_transformer_auto_memmap():
+    """Check that ColumnTransformer works in parallel with joblib's auto-memmapping.
+
+    non-regression test for issue #28781
+    """
+    X = np.random.RandomState(0).uniform(size=(3, 4))
+
+    scaler = StandardScaler(copy=False)
+
+    transformer = ColumnTransformer(
+        transformers=[("scaler", scaler, [0])],
+        n_jobs=2,
+    )
+
+    with parallel_backend("loky", max_nbytes=1):
+        Xt = transformer.fit_transform(X)
+
```

Comment:
CI is failing because this is only doable in joblib>=1.13 and our min is 1.12.
I can make a bigger array for now

Reply:
we could skip the test on that joblib though.